### ISSUE TITLE
Code ligatures

### ISSUE THREAD

**Issue by DeltaRazero** [googlefonts/RobotoMono#39]
Roboto Mono is a font widely-used by people to use in code editors, myself included. Some newer mono fonts for use in code editors contain the addition of ligatures for common programming multi-character combinations, such as [Fira Code](https://github.com/tonsky/FiraCode) and [JetBrains Mono](https://www.jetbrains.com/lp/mono/).

Roboto Mono lacks these ligatures, and as such people have made scripts to merge the ligatures of these fonts into Roboto Mono. However, as they come from a different font altogether, they are quite obviously not in accordance to the style of Roboto Mono. It would be great if official code ligatures could be added.

**Comment by diegoulloao** [comment]
I agree

**Comment by diegoulloao** [comment]
https://github.com/lemeb/a-better-ligaturizer

Roboto Mono with ligatures:
https://github.com/lemeb/a-better-ligaturizer/blob/master/output-fonts/RobotoMono.ttf

Just select the "Book" variant for the Roboto Mono font.

**Comment by DeltaRazero** [comment]
> @DeltaRazero https://github.com/lemeb/a-better-ligaturizer
> 
> Roboto Mono with ligatures: https://github.com/lemeb/a-better-ligaturizer/blob/master/output-fonts/RobotoMono.ttf
> 
> Just select the "Book" variant for the Roboto Mono font.

Yes that works (and that is what I am using right now), but in the end the ligatures are Fira Code's and do not fit Roboto Mono entirely.
